sql/handler.h

```diff
@@ -3835,6 +3838,12 @@ class handler :public Sql_alloc
    notify_table_changed();
  }
 
+/* return false, mean it can't instant alter default */
+virtual bool check_instant_alter(const Alter_inplace_info* inplace_info)
+{
+	return false;
+}
+
```

Comment:
Because we should determine the ALTER_FLAGS in fill_alter_inplace_info() before the check_if_supported_inplace_alter(), so I add this interface check_instant_alter(),